specification/data-semantic-conventions.md

```diff
@@ -14,49 +14,105 @@ This way, the operator will not need to learn specifics of a language and
+
+[HTTP response status code]: https://tools.ietf.org/html/rfc7231#section-6
+[HTTP reason phrase]: https://tools.ietf.org/html/rfc7230#section-3.1.2
+
+It is recommended to also use the general [network attributes][], especially `peer.ip`. If `sock.transport` is not specified, it can be assumed to be `IP.TCP` except if `http.flavor` is `QUIC`, in which case `IP.UDP` is assumed.
+
+### HTTP client
+
+This span type represents an outbound HTTP request.
+
+For a HTTP client span, `SpanKind` MUST be `Client`.
+
+`http.url` is required and represents the HTTP URL used to (initially) make this request.
 
 ## HTTP server
 
 This span type represents an inbound HTTP request.
 
 For a HTTP server span, `SpanKind` MUST be `Server`.
 
-Given an inbound request for a route (e.g. `"/users/:userID?"` the `name`
-attribute of the span SHOULD be set to this route.
+Given an inbound request for a route (e.g. `"/users/:userID?"` the `name` attribute of the span SHOULD be set to this route. If the route does not include the application root path, it SHOULD be prepended to the span name.
 
-If the route can not be determined, the `name` attribute MUST be set to the [RFC 3986 URI](https://www.ietf.org/rfc/rfc3986.txt) path value.
-
-If a framework can identify a value that represents the identity of the request
-and has a lower cardinality than the URI path or route, this value MUST be used for the span name instead.
+If the route can not be determined, the `name` attribute MUST be set as defined in the general semantic conventions for HTTP.
 
 | Attribute name | Notes and examples                                           | Required? |
 | :------------- | :----------------------------------------------------------- | --------- |
-| `component`    | Denotes the type of the span and needs to be `"http"`. | Yes |
-| `http.method` | HTTP request method. E.g. `"GET"`. | Yes |
-| `http.url` | HTTP URL of this request, represented as `scheme://host:port/path?query#fragment` E.g. `"https://example.com:779/path/12314/?q=ddds#123"`. | Yes |
-| `http.route` | The matched route. E.g. `"/users/:userID?"`. | No |
-| `http.status_code` | [HTTP response status code](https://tools.ietf.org/html/rfc7231). E.g. `200` (integer) | No |
-| `http.status_text` | [HTTP reason phrase](https://www.ietf.org/rfc/rfc2616.txt). E.g. `"OK"` | No |
+| `http.target` | The full request target as passed in a [HTTP request line][] or equivalent, e.g. `/path/12314/?q=ddds#123"`. | [1] |
+| `http.host` | The value of the [HTTP host header][]. Note that this might be empty or not present. | [1] |
+| `http.scheme` | The URI scheme identifying the used protocol: `"http"` or `"https"` | [1] |
+| `http.server_name` | The server name (not including port). This should be obtained via configuration. If no such configuration can be obtained, this attribute MUST NOT be set (`host.name` from the [network attributes][] should be used instead). | [1] |
```

Comment:
Do you think `http.server_name` makes sense or should we just use/repurpose `host.name` here?

Reply:
a lot webservers have a name configured in their configuration and I think it's a valuable information to keep this. If you have have several servers running on the same host it's even more important.